ios/brave-ios/Sources/BraveWallet/Crypto/Stores/CryptoStore.swift

```diff
@@ -97,6 +97,7 @@ public class CryptoStore: ObservableObject, WalletObserverStore {
       }
     }
   }
+  @Published var viewActivityToggle: Bool = false
```

Comment:
Did you try moving the `selectedTab` into the store from `CryptoTabsView`? It's confusing have this boolean